internal/compiler-bridge/src/main/scala_2.10/xsbt/Compat.scala

```diff
@@ -76,6 +79,20 @@ abstract class Compat {
     def enclosingTopLevelClass: Symbol = sym.toplevelClass
     def toplevelClass: Symbol = sourceCompatibilityOnly
     def asMethod: MethodSymbol = sym.asInstanceOf[MethodSymbol]
+
+    // Not present in 2.10
+    final def getterIn(base: Symbol): Symbol = sym.getter(base)
+    final def setterIn(base: Symbol, hasExpandedName: Boolean = sym.needsExpandedSetterName): Symbol =
+      sym.setter(base, hasExpandedName)
+
+    // copied from 2.12.1 sources
+    def needsExpandedSetterName: Boolean = (
+      if (sym.isMethod) sym.hasStableFlag && !sym.isLazy
+      else sym.hasNoFlags(LAZY | MUTABLE)
+    )
+
+    // unexpandedName replaces originalName in 2.11
+    def unexpandedName: Name = sym.originalName
```

Comment:
`@inline` here perhaps? Scalac is very pessimistic about inlining, but it's better to have it than not to have it.